src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/ActionPlanValidator.kt

```diff
@@ -14,6 +15,10 @@ class ActionPlanValidator {
 
     validateNumberOfSessionsIsGreaterThanZero(update, errors)
 
+    if (errors.isEmpty()) {
```

Comment:
i understand why this check is here, but if other checks are added to this function and the order is changed we may want to remove this check. perhaps we could group the number of sessions checks together in one method?